cachet_url_monitor/configuration.py

```diff
@@ -54,6 +54,12 @@ def get_current_status(endpoint_url, component_id, headers):
         raise ComponentNonexistentError(component_id)
 
 
+def normalize_url(url):
```

Comment:
Would you mind adding a doc string here? I know the function is self-explanatory, but it's good to have it well documented.
Also, it would be nice to have some unit test.